core/arch/arm/kernel/generic_boot.c

```diff
@@ -614,6 +614,7 @@ static void init_primary_helper(unsigned long pageable_part,
 
 	init_runtime(pageable_part);
 
+	IMSG("\n");
```

Comment:
Please use `IMSG_RAW("\n");` instead, to avoid the prefix.

And this is not the best place to put this, because `init_runtime()` does output some text already when `CFG_WITH_PAGER=y`. Since before `init_runtime()` is too early, you should add one `IMSG_RAW("\n");` in `init_runtime()` before the message "Pager is enabled" (line 240), and another one at the end of the other version of `init_runtime()` (line 428).